shellcheck_gha/console/app.py

```diff
@@ -28,6 +28,17 @@ def parse_args(args: list[str] | None) -> argparse.Namespace:
         action="store_true",
         help="Add debug information (takes precedence over --verbose).",
     )
+    parser.add_argument(
+        "--skip-unknown-files",
+        action=argparse.BooleanOptionalAction,
+        help=(
+            "Whether to exit with an error on when parsing non-GitHub workflow "
+            "or composite action YAML files. Skipping is useful when a directory "
+            "may be mixed with other YAML files "
+            "(e.g. config files such as .github/dependabot.yaml)."
```

Comment:
`--help` is doesn't mention what is the default behavior

```suggestion
            "(e.g. config files such as .github/dependabot.yaml). "
            "Unknown files are skipped by default."
```